### ISSUE TITLE
Product.Variations.GetAll(...)

### ISSUE THREAD

**Issue by alongtheopenroads** [XiaoFaye/WooCommerce.NET#148]
Wordpress version: 4.7.5 
WooCommerce version: 3.0.5
WooCommerce.NET version: 0.7.1

I started using WooCommerce.NET version 0.7.1 and discovered `Product.Variations.GetAll(...)` returns `null` for `sale_price` and `regular_price` even though I have values for those fields in WooCommerce.  However, `sale_price` and `regular_price` do show up in Non-Public members, as shown in the below pic.

```
int parentId = 14538;
var varItems = await wc.Product.Variations.GetAll(parentId);
```
What am I doing wrong?

![image](https://cloud.githubusercontent.com/assets/6580827/26416090/b7352b46-4082-11e7-9a5e-0ccb0b60d438.png)

Thank you,

**Comment by goguvi** [comment]
As I comment in the second point of #147 I think Variation should inherit from JsonObject. This class has a desirialization logic that transform de JSON price value to a decimal c# property.

I make a pull request (#151) with this

**Comment by XiaoFaye** [comment]
This has been fixed in version 0.7.2
